### ISSUE TITLE
Notification stays

### ISSUE THREAD

**Issue by hsrtechs** [robertohuertasm/vscode-icons#295]
**Checklist:**
- I'm using Visual Studio Code Version [1.5.1] [Stable]
- I'm using the extension version [2.0.2]
- [*] I've regarded to the [FAQ](https://github.com/robertohuertasm/vscode-icons/blob/master/README.md#faqs) and they did not help.
- My system is: [OS] [Windows 10]
- [*] I'm sure this issue is not a *duplicate*?
- [*] I crossed out "Description", "Steps to reproduce", "Expected behaviour" and "Actual behaviour".

**Description:**
I'm receiving the notification of what's new in the extension

http://puu.sh/r8r1g/8e2b21a06c.png

**Steps to reproduce:**
1. Installing Extension
2. The above message is there everytime i Restart/Start VSCODE

_Expected behaviour:_

Notification should be there only once after update/install

_Actual behaviour:_

Notification stays there everytime i start/restart

**Comment by robertohuertasm** [comment]
I'm closing this in favour of #293 
Thanks for reporting this! :grinning:
